arviz/tests/test_stats.py

```diff
@@ -134,10 +139,19 @@ def test_summary_nan(centered_eight):
 
 
 def test_summary_bad_fmt(centered_eight):
+    with pytest.raises(TypeError):
```

Comment:
Can these be parameterized? That way they'll fail independently and we'll get an accurate test count

Reply:
Yes